lib/releases.sh

```diff
@@ -49,13 +70,13 @@ BMOBRANCH="${BMOBRANCH:-${BMORELEASE}}"
 
 # On first iteration, jq might not be installed
 if [[ "$CAPIRELEASE" == "" ]]; then
-  command -v jq &> /dev/null && echo "Failed to fetch CAPI release from Github" && exit 1
+  command -v jq &>/dev/null && echo "Failed to fetch CAPI release from Github" && exit 1
 fi
 
 if [[ "$CAPM3RELEASE" == "" ]]; then
-  command -v jq &> /dev/null && echo "Failed to fetch CAPM3 release from Github" && exit 1
+  command -v jq &>/dev/null && echo "Failed to fetch CAPM3 release from Github" && exit 1
 fi
 
 if [[ "$BMORELEASE" == "" ]]; then
-  command -v jq &> /dev/null && echo "Failed to fetch BMO release from Github" && exit 1
+  command -v jq &>/dev/null && echo "Failed to fetch BMO release from Github" && exit 1
 fi
```

Comment:
Why remove the space here? I think it is easier to read with the space so we should add it back.

Reply:
it was aut reformatted by editor linter what do we have coding style for that

Reply:
@tuminoid any preference on this? I'm inclined to approve it as just a matter of preference if there are no objections.

Reply:
That space is completely up to personal taste. I tend to have the space there for readability, but it doesn't really matter.

Reply:
Alright I'm putting lgtm then :slightly_smiling_face: